src/pages/projects/[projectId]/elements/[elementId].tsx

```diff
@@ -1,16 +1,22 @@
 import { Suspense } from "react"
-import { OverallElement, PMElement } from "src/elements/components/ElementDashboard"
 import Head from "next/head"
 import Layout from "src/core/layouts/Layout"
 import getElement from "src/elements/queries/getElement"
 import { useQuery } from "@blitzjs/rpc"
 import { useParam } from "@blitzjs/next"
 import useContributorAuthorization from "src/contributors/hooks/UseContributorAuthorization"
 import { ContributorPrivileges } from "db"
+import { ElementInformation } from "src/elements/components/ElementInformation"
+import { useContributorPrivilege } from "src/contributors/components/ContributorPrivilegeContext"
+import { ElementSummary } from "src/elements/components/ElementSummary"
 
 const ShowElementPage = () => {
+  // Contributor authentication
```

Comment:
I would recommend to be more specific with the language here, as **authentication** is the process for logging in. **Authorization** is the process of checking whether you are permitted to do something.

So you need to be authenticated, in order to get authorization – but it is not guaranteed that you will.